os/arch/arm/src/imxrt/imxrt_pwm.c

```diff
@@ -878,7 +894,31 @@ static int imxrt_pwm_start(FAR struct pwm_lowerhalf_s *dev,
 {
 	FAR struct imxrt_pwmtimer_s *priv = (FAR struct imxrt_pwmtimer_s *)dev;
 
-	imxrt_pwm_starttimer(priv->base, kPWM_Control_Module_0 | kPWM_Control_Module_1 | kPWM_Control_Module_2);
+	printf("info->duty: %d, info->frequency: %d\n", info->duty, info->frequency);
+
+	if (priv->frequency != info->frequency) {
```

Comment:
One question, Duty & Frequency has dependency?? then this is acceptable but, if they works independently, it should changed to if - if instead of if - else if.
I mean if both of duty & frequency is different, then it works properly?